src/Realm.C

```diff
@@ -1928,7 +1942,46 @@ Realm::create_output_mesh()
       if (fileid++ > 0) oname += "-s" + fileid_ss.str();
     }
 
-    resultsFileIndex_ = ioBroker_->create_output_mesh( oname, stk::io::WRITE_RESULTS, *outputInfo_->outputPropertyManager_);
+
+    if(!outputInfo_->catalystFileName_.empty()||
```

Comment:
In theory, we should be able to have multiple blocks without too much issue. For now, we can move with the either/or path and clean up the I/O to allow for a general amount of output blocks. At present, we support one unique output and restart block.